src/main/resources/liquibase/workbench_changelog/20_1_0.xml

```diff
@@ -51,4 +51,43 @@
 		</sql>
 	</changeSet>
 
+	<changeSet author="cuenyad" id="v20.1.0-3">
+		<preConditions onFail="MARK_RAN">
+			  <sqlCheck expectedResult="0">
+                SELECT count(*) FROM workbench_tool WHERE name = 'crop_settings_manager'
+            </sqlCheck>
+        </preConditions>
+		<comment>
+			Add crop settings manager sidebar item.
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			INSERT INTO workbench_tool (`name`, `group_name`, `title`, `version`, `tool_type`, `path`, `parameter`, `user_tool`)
+			VALUES ('crop_settings_manager', 'crop_settings_manager', 'Manage Crop Settings', '1.0', 'WEB', '/ibpworkbench/controller/jhipster#crop-settings-manager', '', 0);
+
+			SET @sidebar_category_id = (SELECT sidebar_category_id FROM workbench_sidebar_category where sidebar_category_name = 'crop_administration');
+
+			INSERT INTO workbench_sidebar_category_link (`tool_name`, `sidebar_category_id`, `sidebar_link_name`, `sidebar_link_title`, `rank`)
+			VALUES ('crop_settings_manager', @sidebar_category_id, 'crop_settings_manager', 'Manage Crop Settings', 2);
+
+			SET @workbench_sidebar_category_link_id =  (SELECT sidebar_category_link_id FROM workbench_sidebar_category_link where tool_name = 'crop_settings_manager');
+			UPDATE PERMISSION SET name = 'MANAGE_CROP_SETTINGS', description = 'Manage Crop Settings', workbench_sidebar_category_link_id = @workbench_sidebar_category_link_id WHERE (name = 'MANAGE_CROP_METADATA');
+		</sql>
+	</changeSet>
+
+	<changeSet author="cuenyad" id="v20.1.0-4">
```

Comment:
@cuenyad I think it is more convenient to have only one changeset in this case. 
Imagine the first one fails, then the second one runs successfully and BMS ends with no Manage metadata neither Manage Crop Settings module